### DISCUSSION TITLE
ORAN 7.2x WG4 Interoperability Setup requirement

### DISCUSSION THREAD

**Discussion by Manimaranchandran** [srsran/srsRAN_Project#324] [General Help]
Hi, We have build up a setup using srsRAN gNB with srsUE, Open5gs. With this, verified basic Sanity until UL & DL Ping reachability.
Any other test cases can be done with existing setup?
Our idea is to have E2E ORAN setup with O-RU, O-DU, O-CU, RIC, SMO. As an initial approach would be focusing on FH 7.2x IOT Test verification. Could you suggest the way forward to keep srsRAN in place and create requirements?
Note: I have checked this setup requirement. Our aim is to verify all the functional, regression using ORAN IOT M-plane spec https://docs.srsran.com/projects/project/en/latest/tutorials/source/oranRU/source/index.html

We would like to have detailed discussion officially too for taking srsRAN in our LAB for ORAN readiness.

Thanks,
Manimaran

**Comment by pgawlowicz** [comment]
Hi, Please contact us directly regarding commercial consulting.
You can find the contact details here: https://srs.io/
If you have since contacted us, please disregard this message.
